install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -29,6 +30,11 @@
 {{- $defaultBpfCtAnyMax = 0 -}}
 {{- end -}}
 
+{{- /* Default values when 1.9 was initially deployed */ -}}
+{{- if semverCompare ">=1.9" (default "1.9" .Values.upgradeCompatibility) -}}
+{{- $defaultBpfTProxy = "true" -}}
+{{- end -}}
+
```

Comment:
@tgraf PTAL, I'm not sure if this is how you intended v1.9 Helm compatibility extension to work.

Reply:
I think we also need to fix https://github.com/cilium/cilium/issues/12288

Reply:
Github is hiding this thread at this point, but this is the remaining piece to figure out /cc @tgraf